Documentation/help_files/message_center_help.php

```diff
@@ -0,0 +1,439 @@
+                <span class="text-right fa-stack fa-lg pull_right small oe-text-black"style="position:relative;right:0;top:0;">
+                    <i class="fa fa-square-o fa-stack-2x"></i>
+                    <i id="print_caret" class="fa fa-caret-up fa-stack-1x"></i>
+                </span>
+                
+                <p><?php echo xlt("The Name cell contains the name, date of birth, patient ID and date of last visit"); ?>.
+                
+                <p><?php echo xlt("You can click on the patient name to quickly access the patient's chart"); ?>.
+                
+                <p><?php echo xlt("The Recall cell has the date of recall and the reason for recall"); ?>.
+                
+                <p><?php echo xlt("The Contacts cell has various bits of contact information like phone numbers, email etc"); ?>.
+                
+                <p><?php echo xlt("The next three cells are the action cells that let you perform some recall activity"); ?>.
+                
+                <p><?php echo xlt("If you check the check box in any of these cells it will generate an entry in the last cell called Progress"); ?>.
+                
+                <p><?php echo xlt("Checking the Postcards check box will let you print a postcard that can be mailed to a patient"); ?>.
+                
+                <p><?php echo xlt("Labels can be printed either for one patient or for all the selected patients"); ?>.
+                
+                <p><?php echo xlt("The Office: Phone cell lets you indicate a phone call was made"); ?>.
+                
+                <p><i class="fa fa-exclamation-circle oe-text-orange" aria-hidden="true"></i> <strong>
+                <?php echo xlt(" The default installation will not make the phone call automatically"); ?>.</strong>
+                
+                <p><?php echo xlt("An appointment can be scheduled by clicking on the calendar icon in that cell"); ?>.
+                <i class="fa fa-calendar-check-o fa-fw oe-text-black" aria-hidden="true"></i>
+                
+                <p><?php echo xlt("Once an appointment is scheduled the recall will drop off the Recall Board"); ?>.
+                
+                <p><?php echo xlt("The Notes cell lets you add a note to an action that you take in the 3 action cells or just a note without any association to the action"); ?>.
+                
+                <p><?php echo xlt("The Progress cell lists all the actions along with any associated notes if any"); ?>.
+                
+                <p><?php echo xlt("You can delete the recall by clicking on the X button on the top right corner of the cell"); ?>.
+            </div>
+            <div class= "row" id="medex_communication_service">
+                <h4 class="oe-help-heading"><?php echo xlt("MedEx Communication Service"); ?><a href="#"><i class="fa fa-arrow-circle-up float-right oe-help-redirect" aria-hidden="true"></i></a></h4>
```

Comment:
Lets drop all the MedEx documentation in the codebase. We don't want to give impression that OpenEMR out of box is not a full release (ie. pay for full feature set).

Reply:
Note there are MedEx and Recall Board (ok to have recall board doc that does not require medex in the codebase) wiki pages.

Reply:
I see your point. I too did not want to convey the impression that a particular paid feature is being promoted officially. Being such a useful feature albeit involving $$ should not a user be aware of the   possibilities?
The beauty of this help page is it is super easy to edit and delete, Simply commenting out the lines will make it disappear from view, with the possibility of reinstatement should one decide to do so in the future. :))

Reply:
What happens when a user activates the MedEx service in Administration > Globals as is suddenly presented with the various sub-menu options on the messages page. It took me plenty of time to figure out that the goodies are associated with $$ . Should I just mention the fact about the service  and refer user to the MedEx webpage which unfortunately does not have good documentation.

Reply:
Even commented out code will go to the translation pipeline though (ie. translators would be stuck translating unnecessary stuff). Could do a very brief mention of it and then direct to the current MedEx page on the wiki.

Reply:
We could copy/paste your MedEx documentation into that wiki page; guessing Ray would be very happy with that.

Reply:
Will remove the info. and point to the wiki page. I hope that the wiki page is updated otherwise we would again be in a situation where we have a brand new feature with no easy way to figure out how to use it.
BTW how much MedEx info  do you want me to leave on the page?